src/main/java/helpers/WebElementHelper.java

```diff
@@ -45,4 +45,8 @@ public static boolean areVisible(WebElement... elements) {
     }
     return elementsNotVisible.isEmpty();
   }
+
+  public static String getTextFromPage(WebElement element){
```

Comment:
do we need the `FromPage` part? maybe use `getElementText()` to be more accurate?